OpenHAB.Windows/Controls/SetpointWidget.xaml.cs

```diff
@@ -20,7 +21,7 @@ public SetpointWidget()
 
         private void ButtonUp_Click(object sender, RoutedEventArgs e)
         {
-            double value = double.Parse(Widget.Item.State);
+            double value = getDoubleValue(Widget.Item.State);
```

Comment:
@cd-tronic Please rename the function to start in upper case (C# Naming Guidlines).  
getDoubleValue -> GetDoubleValue